WordPress/gutenberg
# URL: Use test data from web-platform-tests for isURL spec conformance

Closes #20534

This pull request seeks to enhance tests for `@wordpress/url` `isURL` to conform to those defined by the [Web Platform Tests project](https://github.com/web-platform-tests/wpt). As outlined in #20534, it includes a script to download and transform the [URL test data](https://github.com/web-platform-tests/wpt/blob/master/url/resources/urltestdata.json), which is then loaded by the `@wordpress/url` unit tests as fixture data for the `isURL` tests.

You can run the script directly:

```
node packages/url/scripts/download-wpt-data.js
```

**Testing Instructions:**

Ensure unit tests pass:

```
npm run test-unit packages/url/src/test/index.test.js
```
